LICENSE.md

```diff
@@ -27,6 +27,10 @@ for exceptions.
 > OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
 > WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
+Julia includes code from the following projects, which have their own licenses:
+- [LDC](https://github.com/ldc-developers/ldc/blob/master/LICENSE) (for ccall/cfunction ABI definitions)
+- [MUSL](http://git.musl-libc.org/cgit/musl/tree/COPYRIGHT) (for getopt implementations on Windows)
+- [NetBSD](http://www.netbsd.org/about/redistribution.html) (for setjmp/longjmp implementations on Windows)
```

Comment:
Yes, but that information isn't very useful, except for people owning the copyright for most of the rest of the Julia codebase and who may want to relicense it (which likely isn't going to happen). Is it really needed here, rather than in the file header?

Anyway, if it's equivalent to MIT, it should be written explicitly.